README.md

```diff
@@ -53,6 +53,41 @@ that you might have to run sometimes.
  - `git-fatlas-remake-package-list`: force rebuilding the package
    list. If you add or remove a bunch of things you might have to call
    this. Hopefully not though.
+   
```

Comment:
In the spirit of your MR, maybe we should add a line after this saying something like "for most other commands you can use standard git"